src/main/ui/app/component/AddOnSearch.jsx

```diff
@@ -57,31 +57,40 @@ export default class AddOnSearch extends Component {
 
     render() {
         return (
-                <div>
-                    Type:
-                    <select onChange={event => this.setType(event.target.value)}>
-                        <option value="">All Types</option>
-                        <option value="OMOD">Module (OMOD)</option>
-                        <option value="OWA">Open Web App (OWA)</option>
-                    </select>
-
-                    <br/>
+                <div className="row col-md-12 col-sm-12 col-xs-12 pushdown">
+                    <div>
+    <div className="input-group">
+    
+      <div className="input-group-btn">
+        <button type="button" className="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Type <span className="caret"></span></button>
```

Comment:
I think you need `<DropdownButton>` for this: https://react-bootstrap.github.io/components.html#btn-dropdowns